samples/wifi/sr_coex/README.rst

```diff
@@ -71,90 +71,28 @@ Configuration options
 
 The following sample-specific Kconfig options are used in this sample (located in :file:`samples/wifi/sr_coex/Kconfig`):
 
-.. _CONFIG_COEX_SEP_ANTENNAS:
-
-CONFIG_COEX_SEP_ANTENNAS
-   This option specifies whether the antennas are shared or separate for Bluetooth and WLAN.
-
-.. _CONFIG_TEST_TYPE_WLAN_ONLY:
-
-CONFIG_TEST_TYPE_WLAN_ONLY
-   This option enables the WLAN test type.
-
-.. _CONFIG_TEST_TYPE_BLE_ONLY:
-
-CONFIG_TEST_TYPE_BLE_ONLY
-   This option enables the Bluetooth LE test type.
-
-.. _CONFIG_TEST_TYPE_WLAN_BLE:
-
-CONFIG_TEST_TYPE_WLAN_BLE
-   This option enables concurrent WLAN and Bluetooth LE tests.
-
-.. _CONFIG_WIFI_TEST_DURATION:
-
-CONFIG_WIFI_TEST_DURATION
-   This option sets the Wi-Fi test duration in milliseconds.
-
-.. _CONFIG_BLE_TEST_DURATION:
-
-CONFIG_BLE_TEST_DURATION
-   This option sets the Bluetooth test duration in milliseconds.
-
-.. _CONFIG_INTERVAL_MIN:
-
-CONFIG_INTERVAL_MIN
-   This option sets the Bluetooth minimum connection interval (each unit is 1.25 milliseconds).
-
-.. _CONFIG_INTERVAL_MAX:
-
-CONFIG_INTERVAL_MAX
-   This option sets the Bluetooth maximum connection interval (each unit is 1.25 milliseconds).
-
-.. _CONFIG_STA_SAMPLE_SSID:
-
-CONFIG_STA_SAMPLE_SSID
-   This option specifies the SSID to connect.
-
-.. _CONFIG_STA_SAMPLE_PASSWORD:
-
-CONFIG_STA_SAMPLE_PASSWORD
-   This option specifies the passphrase (WPA2) or password WPA3 to connect.
-
-.. _CONFIG_STA_KEY_MGMT_*:
-
-CONFIG_STA_KEY_MGMT_*
-   These options specify the key security option.
-
-.. _CONFIG_BT_THROUGHPUT_FILE:
-
-CONFIG_BT_THROUGHPUT_FILE
-   This option selects the type of the throughput test.
-
-.. _CONFIG_BT_THROUGHPUT_DURATION:
-
-CONFIG_BT_THROUGHPUT_DURATION
-   This option sets the Bluetooth throughput test duration in milliseconds.
+.. options-from-kconfig::
+   :show-type:
 
 Configuration files
 ===================
 
 To enable different test modes, set up the following configuration parameters in the :file:`prj.conf` file:
 
-* Antenna configuration: Use the :ref:`CONFIG_COEX_SEP_ANTENNAS <CONFIG_COEX_SEP_ANTENNAS>` Kconfig option to select the antenna configuration.
+* Antenna configuration: Use the :kconfig:option:`CONFIG_COEX_SEP_ANTENNAS` Kconfig option to select the antenna configuration.
   Set it to ``y`` to enable separate antennas and ``n`` to enable shared antenna.
 * Test modes: Use the following Kconfig options to select the required test case:
 
-  * :ref:`CONFIG_TEST_TYPE_WLAN_ONLY <CONFIG_TEST_TYPE_WLAN_ONLY>` for Wi-Fi only test
-  * :ref:`CONFIG_TEST_TYPE_BLE_ONLY <CONFIG_TEST_TYPE_BLE_ONLY>` for Bluetooth LE only test
-  * :ref:`CONFIG_TEST_TYPE_WLAN_BLE <CONFIG_TEST_TYPE_WLAN_BLE>` for concurrent Wi-Fi and Bluetooth LE test.
+  * :kconfig:option:`CONFIG_TEST_TYPE_WLAN_ONLY` for Wi-Fi only test
```

Comment:
can you please move these changes to a separate commit?

Reply:
Build errors were seen related to the `` :ref:`` label after using ``:show_type:`` extension. So, addressed the issues by making all the necessary changes in a single commit.